packages/gatsby/src/bootstrap/load-plugins/index.ts

```diff
@@ -88,9 +88,7 @@ export async function loadPlugins(
   const config = normalizeConfig(rawConfig)
 
   // Show errors for invalid plugin configuration
-  if (process.env.GATSBY_EXPERIMENTAL_PLUGIN_OPTION_VALIDATION) {
-    await validateConfigPluginsOptions(config, rootDir)
-  }
+  await validateConfigPluginsOptions(config, rootDir)
```

Comment:
@mxstbr Isn't this technically a breaking change for all plugins when they remove the feature flag? 🤔
Since these plugins won't work anymore with older `gatsby` versions, because it now errors out with the message to upgrade.

So if that's correct, the peerDependency of all plugins that remove the feature flag should be updated to `gatsby@^2.25.0` (which is a breaking change) or am I forgetting something?

Reply:
@mxstbr Any thoughts on this? 🤔

Reply:
@mxstbr Any thoughts on this? 🤔